Blog post introducing Ronny, Joel and Nick :-) (#1299)
ropensci/roweb3: content/blog/2026-05-26-new-editors/index.md

---
slug: neweditors2026
title: ''
author:

>     - 

Author: ```suggestion
    - Ronny Hernandez Mora
```

>     - 

Author: ```suggestion
    - Joel Nitta
    - Nicholas Tierney
```

    - Yanina Bellini Saibene
editor:
    - Mark Padgham
date: '2026-05-26'
tags:
  - Software Peer Review
  - editors
  - rOpenSci team
featured: true
description: Introducing three new editors for rOpenSci software peer review
params:
  doi: 
---

Meet our new editors!

> ## 

Author: ```suggestion
## Ronny Hernandez Mora
```

{{< imgtxt src = "/img/team/" width = "200" alt = "Headshot of ">}}

{{< /imgtxt >}}

xx on [GitHub](), [Website]().

{{< quote footer="name" >}}

Quote

{{< /quote >}}

> ## TODO: add the other people

Author: ```suggestion

## Joel Nitta

## Nicholas Tierney
```

---
slug: neweditors2026

> title: ''

Author: ```suggestion
title: 'Ronny Hernandez Mora, Joel Nitta, and Nicholas Tierney Join rOpenSci Software Peer Review Editorial Team'
```

author:
    - Ronny Hernandez Mora
    - Joel Nitta
    - Nicholas Tierney
    - Yanina Bellini Saibene
editor:
    - Mark Padgham
date: '2026-05-26'
tags:
  - Software Peer Review
  - editors
  - rOpenSci team
featured: true
description: Introducing three new editors for rOpenSci software peer review
params:
  doi: 
---

> Meet our new editors!

Author: ```suggestion
What makes [rOpenSci's software peer review](/software-review/) work is people who care deeply about the quality and usability of scientific software, and who give their time and expertise to help others build it better. Today, we're glad to share that three such people are joining our editorial team.

We welcome *Ronny Hernandez Mora*, *Joel Nitta*, and *Nicholas Tierney* as new editors. Each brings a distinct perspective shaped by their work, their communities, and their experience with open-source R development. Together, they strengthen our capacity to serve the growing number of package authors who submit their work for review, and to uphold the collaborative, friendly and transparent standards that rOpenSci software peer review is known for.
```

## Ronny Hernandez Mora

{{< imgtxt src = "/img/team/" width = "200" alt = "Headshot of ">}}

{{< /imgtxt >}}

xx on [GitHub](), [Website]().

{{< quote footer="name" >}}

Quote

{{< /quote >}}

## Joel Nitta

## Nicholas Tierney

## About the Software Peer Review Program

rOpenSci's software peer review program brings together volunteers to collaboratively review scientific and statistical software according to transparent, constructive, and open standards. 
We have two different type of review: one for [general research software packages](https://devguide.ropensci.org/) and another for [package that implement statistical methods](https://stats-devguide.ropensci.org). 

> Editors manage submissions, coordinate reviewers, and help guide packages through review to improve code quality, documentation, and usability. 

Author: ```suggestion
Editors like Ronny, Joel, and Nick are central to that process: they handle initial submission checks, identify and coordinate reviewers, and guide authors through the review until their package is ready.
```

## Get Involved

> Are you considering submitting your package for review? These resources will help:

Author: ```suggestion
Thinking about submitting a package? Start here:
```

> - About [rOpenSci Software Peer Review](/software-review/);

Author: ```suggestion
- [rOpenSci Software Peer Review](/software-review/): scope, process, and guidelines;
```

> - Read public [software review threads on GitHub](https://github.com/ropensci/software-review/issues)

Author: ```suggestion
-  [rOpenSci Packages: Development, Maintenance, and Peer Review](https://github.com/ropensci/software-review/issues): the full guide for package authors;
-  [rOpenSci Statistical Software Peer Review](https://stats-devguide.ropensci.org): the full guide for statistical software submissions;
- Public [software review threads on GitHub](https://github.com/ropensci/software-review/issues): see software peer review in action.
```

> - Fill out the [rOpenSci Reviewer Sign-Up Form](https://airtable.com/app8dssb6a7PG6Vwj/shrnfDI2S9uuyxtDw) to volunteer to review.

Author: ```suggestion
- Would you like to contribute as a reviewer? We'd love to have you. Fill out the [rOpenSci Reviewer Sign-Up Form](https://airtable.com/app8dssb6a7PG6Vwj/shrnfDI2S9uuyxtDw) and we'll match you with packages that fit your expertise.
```

> Welcome again ** **! We're excited to have you join the editorial team.

Author: ```suggestion
A warm welcome to **Ronny, Joel, and Nick**! We're very glad to have you with us. 
```

author:
    - Ronny Hernandez Mora
    - Joel Nitta
    - Nicholas Tierney
    - Yanina Bellini Saibene
editor:
    - Mark Padgham
date: '2026-05-26'
tags:
  - Software Peer Review
  - editors
  - rOpenSci team
featured: true
description: Introducing three new editors for rOpenSci software peer review
params:
  doi: 
---

What makes [rOpenSci's software peer review](/software-review/) work is people who care deeply about the quality and usability of scientific software, and who give their time and expertise to help others build it better. Today, we're glad to share that three such people are joining our editorial team.

We welcome *Ronny Hernandez Mora*, *Joel Nitta*, and *Nicholas Tierney* as new editors. Each brings a distinct perspective shaped by their work, their communities, and their experience with open-source R development. Together, they strengthen our capacity to serve the growing number of package authors who submit their work for review, and to uphold the collaborative, friendly and transparent standards that rOpenSci software peer review is known for.

## Ronny Hernandez Mora

> {{< imgtxt src = "/img/team/" width = "200" alt = "Headshot of ">}}

Author: ```suggestion
{{< imgtxt src = "/img/team/ronny-hernandez-mora.png" width = "200" alt = "Headshot of Ronny">}}

```

{{< /imgtxt >}}

xx on [GitHub](), [Website]().

> {{< quote footer="name" >}}

Author: ```suggestion
{{< quote footer="Ronny" >}}
```

> Quote

Author: ```suggestion
TODO: Add Ronny's Quote about rOpenSci
```

---
slug: neweditors2026

> title: 'Ronny Hernandez Mora, Joel Nitta, and Nicholas Tierney Join rOpenSci Software Peer Review Editorial Team'

Author: ```suggestion
title: 'Ronny Hernandez Mora, Joel Nitta, and Nick Tierney Join rOpenSci Software Peer Review Editorial Team'
```

author:
    - Ronny Hernandez Mora
    - Joel Nitta
    - Nicholas Tierney
    - Yanina Bellini Saibene
editor:
    - Mark Padgham
date: '2026-05-26'
tags:
  - Software Peer Review
  - editors
  - rOpenSci team
featured: true
description: Introducing three new editors for rOpenSci software peer review
params:

>   doi: 

Author: ```suggestion
  doi: 10.59350/f2n2n-4y807
```

---

> What makes [rOpenSci's software peer review](/software-review/) work is people who care deeply about the quality and usability of scientific software, and who give their time and expertise to help others build it better. Today, we're glad to share that three such people are joining our editorial team.

Reviewer: ```suggestion
What makes [rOpenSci's software peer review](/software-review/) work is people who care deeply about the quality and usability of scientific software, and who give their time and expertise to help others build it better. Today, we're pleased to announce three new members of our editorial team.
```

> We welcome *Ronny Hernandez Mora*, *Joel Nitta*, and *Nicholas Tierney* as new editors. Each brings a distinct perspective shaped by their work, their communities, and their experience with open-source R development. Together, they strengthen our capacity to serve the growing number of package authors who submit their work for review, and to uphold the collaborative, friendly and transparent standards that rOpenSci software peer review is known for.

Reviewer 2: Hi @yabellini! Thanks for putting this together. I have one clarification about my inclusion in the post: I started as an editor in August 2025, so I’m not exactly “joining” now as a new editor.

Could we adjust the wording so that it does not imply that I'm starting this month? I care about this because I list my editor role as starting in August 2025, and I’d like the public rOpenSci record to be consistent with that.

Maybe we could use something like:

“We welcome Joel Nitta and Nicholas Tierney as new editors, and formally introduce Ronny Hernandez Mora, who joined the editorial team in August 2025.”

Author: Yes, of course! Thank you for the clarification.

> We welcome *Ronny Hernandez Mora*, *Joel Nitta*, and *Nicholas Tierney* as new editors. Each brings a distinct perspective shaped by their work, their communities, and their experience with open-source R development. Together, they strengthen our capacity to serve the growing number of package authors who submit their work for review, and to uphold the collaborative, friendly and transparent standards that rOpenSci software peer review is known for.

Reviewer: ```suggestion
We welcome *Joel Nitta*, and *Nicholas Tierney* as new editors, and formally introduce *Ronny Hernandez Mora*, who joined the editorial team in August 2025. Each brings a distinct perspective shaped by their work, their communities, and their experience with open-source R development. Together, they strengthen our capacity to serve the growing number of package authors who submit their work for review, and to uphold the collaborative, friendly and transparent standards that rOpenSci software peer review is known for.
```

## Ronny Hernandez Mora

{{< imgtxt src = "/img/team/ronny-hernandez-mora.png" width = "200" alt = "Headshot of Ronny">}}

TODO: add Ronny's bio
{{< /imgtxt >}}

xx on [GitHub](), [Website]().

{{< quote footer="Ronny" >}}

TODO: Add Ronny's Quote about rOpenSci

{{< /quote >}}

## Joel Nitta
{{< imgtxt src = "/img/team/joel-nitta.png" width = "200" alt = "Headshot of Joel">}}

TODO: Add Joel's bio.

{{< /imgtxt >}}

Joel on [GitHub](), [Website]().

{{< quote footer="Joel" >}}

TODO: Add Joel's quote about rOpenSci

{{< /quote >}}
## Nicholas Tierney
{{< imgtxt src = "/img/team/nick-tierney.jpg" width = "200" alt = "Headshot of ">}}

> TODO: Add Nick's bio.

Reviewer 3: ```suggestion
Nicholas (Nick) Tierney is a statistician, [Research Software Engineer](https://researchsoftware.org/), and freelance consultant with a PhD in Statistics who specialises in data analytics, R package development, and teaching. Previously, he worked with [Professor Nick Golding](https://www.thekids.org.au/contact-us/our-people/g/nick-golding/) at [The Kids Research Institute Australia](https://www.thekids.org.au/) and was a Research Fellow at Monash University with [Professor Dianne Cook](https://www.dicook.org/), where he developed tools for exploratory data analysis including [visdat](https://docs.ropensci.org/visdat/), [naniar](https://naniar.njtierney.com/), and [brolgar](https://brolgar.njtierney.com/). Nick actively writes about R related projects at his blog, ["credibly curious"](https://www.njtierney.com/). When not coding, Nick enjoys outdoor adventures and hiked the entire Pacific Crest Trail in 2023, documenting his journey at [njt.micro.blog](https://njt.micro.blog/).
```

{{< /imgtxt >}}

> Nick on [GitHub](), [Website]().

Reviewer 3: ```suggestion
Nick on [GitHub](https://github.com/njtierney), [Website](https://www.njtierney.com/).
```

{{< quote footer="Nick" >}}

> TODO: Add Nick's quote about rOpenSci

Reviewer 3: ```suggestion
I remember seeing rOpenSci online in 2014, at their first [rOpenSci hackathon](https://unconf14.ropensci.org/) in San Francisco, USA. I was inspired by not just the projects they worked on, but the collective of people who were kind and generous as much as they were brilliant. I helped run an offshoot of the rOpenSci Unconf in Australia, which ran in [2016](https://auunconf.ropensci.org/), [2017](https://ozunconf17.ropensci.org/), [2018](https://ozunconf18.ropensci.org/), and [2019](https://ozunconf19.ropensci.org/). I received amazing mentorship and support from Karthik Ram and Stefanie Butland in learning how to run these unconferences. I think these have had a lasting impact on connecting people from Australia and New Zealand. 

I had my R package, [visdat](https://docs.ropensci.org/visdat/), reviewed by rOpenSci in 2017, and this experience was formative to my understanding of software review, and greatly improved my own day to day practice. In 2024, with [Eric Scott](https://github.com/Aariq), and [Andrew Brown](https://github.com/brownag), we submitted the [geotargets](https://github.com/ropensci/geotargets) package to rOpenSci. 

rOpenSci has always been at the forefront of building a community of practice, and their model of peer review is a gold standard. Science would be better if more journals practised such transparency and kindness. It is an enormous honour to be able to give back to a community that has been so formative in my career and my life.
```
